models.py

```diff
@@ -15,7 +15,7 @@
 
 class Node(db.Model):
     __tablename__ = 'node'
-    node_key = db.Column(db.String(34), primary_key=True)
+    node_key = db.Column(db.String(42), primary_key=True)
```

Comment:
I think length was ok
But session ID length has problems, @ignasbernotas ?

Reply:
Not sure about session ID (it has a separate Jira task), but`node_key` length is not ok, here is an example of `node_key` from running node:
`0xdD944d65b963cEe10DbF0a7a3dc8c8355f7F1Dc9`
This id has a length of 42. BTW, https://simple.wikipedia.org/wiki/42_(answer) :D

Reply:
Yup it's 42:D